### PR TITLE
Tinloof/2px rating

### PR DESCRIPTION
**Pull request by freadbaker**
Implements the 2px vibe's Rating component.

### CODE REVIEW

**Review by migueloller — APPROVED** [review]
Looks like this PR has a bunch of changes that are also included in other PRs. Maybe we should go ahead and merge to the `2px` branch so we can consolidate and have other smaller PRs that focus on adding 1 component at a time?

File: apps/web/vibes/2px/components/icons/index.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      <path
+        className="fill-foreground"
+        stroke="currentColor"
+        d="M286.897 389.42V119.426l-51.471 111.586-122.029 14.469 90.219 83.431-23.949 120.529 107.23-60.021Z"
+      />
+
+      {/* Star stroke */}
+      <path
+        stroke={props.stroke || 'currentColor'}
+        stroke-width="50.324"
```
Comment (LHDi) [inline apps/web/vibes/2px/components/icons/index.tsx:19]: `strokeWidth` @freadbaker

File: apps/web/vibes/2px/components/icons/index.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      <path
+        className="fill-foreground"
+        stroke="currentColor"
+        d="M286.897 389.42V119.426l51.471 111.586 122.029 14.469-90.219 83.431 23.949 120.529-107.23-60.021Z"
+      />
+
+      {/* Star stroke */}
+      <path
+        stroke={props.stroke || 'currentColor'}
+        stroke-width="50.324"
```
Comment (LHDi) [inline apps/web/vibes/2px/components/icons/index.tsx:52]: `strokeWidth`

File: apps/web/vibes/2px/components/icons/index.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      xmlns="http://www.w3.org/2000/svg"
+      fill="none"
+      viewBox="0 0 575 547"
+      {...props}
+      stroke="none"
+    >
+      {/* Star stroke */}
+      <path
+        stroke={props.stroke || 'currentColor'}
+        stroke-width="50.324"
```
Comment (LHDi) [inline apps/web/vibes/2px/components/icons/index.tsx:73]: `strokeWidth`

**Review by LHDi — APPROVED** [review]
LGTM

Just some minor changes needed @freadbaker

**Pull request merged**